docs/source/index.rst

```diff
@@ -100,7 +102,7 @@ The following are the four main endpoints that assist with the high-level goals
 Manifest Generation
 ~~~~~~~~~~~~~~~~~~~
 
-Provides a manifest template for users for a particular project or data type. If a project with annotations already exists, a semi-filled-out template is provided to the user so that they do not start from scratch. If there are no existing annotations, an empty manifest template is provided.
+Provides a manifest template for users for a particular project or data type. If a project with annotations already exists, a semi-filled-out template can be provided to the user. This ensures they do not start from scratch. If there are no existing annotations and manifests, an empty manifest template is provided.
```

Comment:
@thomasyu888 I modified this section here because: 
*  "If a project with annotations already exists, a semi-filled-out template is provided to the user so that they do not start from scratch." might be a bit confusing. Schematic has "use_annotations" parameter that let people control if they want to generate a manifest and pull annotations. 
* if there's existing manifests, schematic would use that manifest as well so not just annotations.